okhttp/src/main/kotlin/com/connectrpc/okhttp/OkHttpStream.kt

```diff
@@ -175,7 +172,8 @@ private class ResponseCallback(
     }
 }
 
-internal class PipeDuplexRequestBody(
+internal class PipeRequestBody(
+    private val duplex: Boolean,
```

Comment:
This was necessary in order to do client or server streaming over HTTP 1.1.

Reply:
Nice! I expected this might be a bit more tricky - especially for client streaming calls.